docs/UserGuide.md

```diff
@@ -218,153 +215,148 @@ Scroll through your history of commands using the `↑` or `↓` arrow keys.
+`edit INDEX [k/KEYWORDS]...`
+* Information of the contact at your specified `INDEX` will be modified. 
+* The index refers to the index number shown on the displayed person list. 
 * The index **must be a positive integer** 1, 2, 3, …​
-* At least one of the optional fields must be provided.
-* Users can opt to change any fields associated with a contact as long as they prepend the argument with the corresponding flags according to the format above.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tag after it.
-* When editing dates, all existing dates of the person will be removed i.e adding of dates is not cumulative.
-* You can remove all the person’s dates by typing `date/` without
-  specifying any date after it.
+* You can provide the following fields (i.e. `[k/KEYWORDS]`) in any order when editing a contact's information: `[n/NAME] [p/PHONE] [r/REMARK] [t/TAG]…​ [date/NAME:YYYY-MM-DD[:monthly|:yearly]]…​ [fb/FACEBOOK] [ig/INSTAGRAM] [tele/TELEGRAM] [tiktok/TIKTOK] [twitter/TWITTER]` 
+* You should provide at least one field from the above list for this command to run successfully.
+* In addition, you can add a name for each `DATE` (e.g. Birthday, Meet-Up, Anniversary).
+* If your dates represent events that recur monthly or yearly, you can add the keyword `:monthly` or `:yearly` behind the date which has to be specified in `YYYY-MM-DD` format.
+* Furthermore, upcoming dates (within 7 days) will be highlighted in the user interface.
 </div>
 
-<div markdown="block" class="alert alert-success">
-**:heavy_check_mark: Example:** `edit 1 p/91234567 fb/Yalex19` 
+<div markdown="block" class="alert alert-warning">:exclamation: **Caution:**
+
+* Before you specify information for any field that you wish to modify, you should prepend your input with its corresponding flag (e.g. `n/` `p/` etc.) according to the format above.
+* You should check your input before you hit ENTER as SociaLite will overwrite the contact's existing information for any valid field that you furnish.
+* When you modify `TAG`s or `DATE`s, the existing `TAG`s or `DATE`s of the person will be removed i.e. adding of `TAG`s or `DATE`s is not cumulative.
+* Your input should conform with the character limit for the following fields:
+
+    | Field    | Character Limit |
+    | `TAG`    | 50              |
+    | `DATE`   | 50              |
+    | `REMARK` | 150             |
 
-Edits the phone number and Facebook handle of the 1st person to be `91234567` and `Yalex19` respectively.
-![05_edit](images/UG/05_edit.png)
 </div>
 
 <div markdown="block" class="alert alert-success">
-**:heavy_check_mark: Example:** `edit 4 n/Betsy Crower t/`
+**:heavy_check_mark: Example:** Edit `NAME`
```

Comment:
I think the grave accent on this line, line 258 and line 266 can be a bit confusing to users! Maybe we can just get rid of the inline code and use "Edit phone number" "Edit contact name" "Edit a tag" instead.